frontend/src/pages/LandingPage/style.css

```
@@ -1,12 +1,12 @@
 /* --- LANDING SPECIFIC STYLES --- */
 .landing-page {
-    background-color: #000;
-    color: #f7f7f8;
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
+    background-color: #030303;
+    color: #fcfcfc;
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)